### PR TITLE
Update install page

### PR DESCRIPTION
**Pull request by andraxylia**
Fixes istio/issues#64
Fixes istio/issues#63

### CODE REVIEW

File: _docs/setup/kubernetes/quick-start.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ Quick Start instructions to install and configure Istio in a Kubernetes cluster.
 
 The following instructions assume you have access to a Kubernetes **1.7.4 or newer** cluster
 with [RBAC (Role-Based Access Control)](https://kubernetes.io/docs/admin/authorization/rbac/) enabled.
+If you wish to enable transparent injection of sidecar, you need to turn on Kubernetes alpha features in your cluster.
```
Comment (rshriram) [inline _docs/setup/kubernetes/quick-start.md]: Add a link to the automatic-sidecar-inject.html page

Comment (ldemailly) [reply _docs/setup/kubernetes/quick-start.md]: there is already a " If your cluster has Kubernetes alpha features enabled,  ..." below - do we need to turn people away that don't have alpha features ?

Comment (andraxylia) [reply _docs/setup/kubernetes/quick-start.md]: done, ptal

**Pull request merged**